Content.Server/Cloning/CloningSystem.cs

```diff
@@ -280,6 +289,18 @@ public override void Update(float frameTime)
             }
         }
 
+        // On emag, spawns a failed clone when cloning process fails which attacks nearby crew.
+        private void OnEmagged(EntityUid uid, CloningPodComponent clonePod, ref GotEmaggedEvent args)
+        {
+            if (_powerReceiverSystem.IsPowered(uid))
+            {
+                EnsureComp<EmaggedComponent>(uid);
+                _audio.PlayPvs(clonePod.SparkSound, uid, AudioParams.Default.WithVolume(8));
+                _popupSystem.PopupEntity(Loc.GetString("cloning-pod-component-upgrade-emag-requirement"), uid);
+                args.Handled = true;
+            }
+        }
```

Comment:
1. Invert the if and use this.IsPowered
2. Shouldn't need to EnsureComp
3. The AudioParams should be on the sound.